composites/basic/Table/ListTable.js

```diff
@@ -75,7 +74,9 @@ class ListTable extends React.Component {
 	render() {
 		let children = this.getChildren();
 
-		return ( <List role="list" children={ children } /> );
+		return (
+			<List>{ children }</List>
```

Comment:
Can we keep `role="list"` please? Without that, Safari and VoiceOver don't announce the list as a list. We're already doing that in other places.